composer.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "silverstripe/theme-fontpicker",
+    "description": "Customise site fonts from the admin settings",
+    "type": "silverstripe-vendormodule",
+    "keywords": [
+        "silverstripe"
+    ],
+    "license": "BSD-3-Clause",
+    "require": {
+        "silverstripe/framework": "^4.1",
```

Comment:
It would make more sense to target a specific version of admin and/or siteconfig.